notebooks/alibabacloud-ai-search/inference-alibabacloud-ai-search.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
_Commented on notebook `notebooks/alibabacloud-ai-search/inference-alibabacloud-ai-search.ipynb` Cell 2 Line 5_
```markdown
# Requirements

For this example, you will need:

- An Elastic deployment with minimum **4GB machine learning node**
```
We use Alibaba's Cloud right? So we dont need a ML Node here as the work is proxied through to Alibaba

Reply:
Thank you for your suggestion! I have removed the restriction on the 4GB machine learning node.